src/main/resources/mybatis-mapper/customer-mapper.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+
+<!DOCTYPE mapper
+        PUBLIC "-//mybatis.org//DTD Mapper 3.0//EN"
+        "http://mybatis.org/dtd/mybatis-3-mapper.dtd">
+
+<mapper namespace="com.solvd.db.dao.interfaces.ICustomerDAO">
+
+    <resultMap id="CustomerResultMap" type="Customer" autoMapping="false">
+        <id property="customerId" column="customer_id"/>
+        <association property="person" column="person_id" javaType="Person"
+                     select="com.solvd.db.dao.interfaces.IPersonDAO.getById"/>
+        <collection property="accounts" column="account_number" ofType="Account" javaType="ArrayList"
+                    resultMap="com.solvd.db.dao.interfaces.IAccountDAO.AccountResultMap" />
+    </resultMap>
```

Comment:
Is this the right way to mention collections? In this case, i only get the account number in the list but rest of the information about the account is not present. For example
Customer{customerId=4
, person=Person{personId=6, firstName='person6', lastName='person6last', dateOfBirth=1975-11-30, phoneNumber='1231231234', email='person6@abcd.com', address=Address{addressId=6, line1='777 xyz dr', line2='null', zipCode='77777', city=City{cityId=6, cityName='bombay', country=Country{countryId=6, countryName='india'}}}}
, accounts=[Account{
         accountNumber=20230005, balance=0.0, minBalance=0.0, accountType=null}]
}